cluster-operator/src/main/java/io/strimzi/operator/cluster/model/FromCrdUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.strimzi.api.kafka.model.template.PodDisruptionBudgetTemplate;
+import io.strimzi.api.kafka.model.template.PodTemplate;
+
+/**
+ * This class contains static methods which are used commonly for parsing CRDs into Models
+ */
+public class FromCrdUtils {
```

Comment:
We already have `ModelUtils`, so what is your thinking in creating another class with `static` methods?

Reply:
My logic was that I would love to avoid having a huge `ModelUtils` class with many unrelated methods. And these two methods seemed like something what can be very easily factored out into separate class - they are used only from a very specific place, the `fromCrd` methods of the model classes. So I created the `FromCrdUtils` class.

But if you think they should be in the `ModelUtils` I can move them, it should be quite easy.